db/view/view.cc

```diff
@@ -37,6 +37,7 @@
 #include "db/view/view_builder.hh"
 #include "db/view/view_updating_consumer.hh"
 #include "db/view/view_update_generator.hh"
+#include "db/view/regular_column_transformation.hh"
```

Comment:
A 'computed' repetition in the commit's msg: "This new type of computed computed "